src/components/UserManagement/ResetPasswordUserPopup.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
```

Comment:
What's the difference between this file and the old ResetPasswordPopup?

Reply:
Removed this file, this is the wrong one which is accidentally included.